front/shared/input_bar/InputBarContainer.tsx

```diff
@@ -162,7 +166,7 @@ const InputBarContainer = ({
                   editorService.insertMention({ id: c.sId, label: c.name });
                 }}
                 assistants={allAssistants}
-                showFooterButtons={true}
+                showFooterButtons={false} // TODO(EXTENSION) disabled footer button here
```

Comment:
???

Reply:
footerButtons is "create assistant", which can't be used in extension